### PR TITLE
Feature/context logger shorthands

### PR DESCRIPTION
**Pull request by xaionaro**

### CODE REVIEW

**Review by mimir-d — DISMISSED** [review]

File: context.go [hunk 1]
```diff
@@ -0,0 +1,110 @@
+// Copyright 2022 Meta Platforms, Inc. and affiliates.
```
Comment (u1f35c) [inline context.go]: 2023.

File: internal/context.go [hunk 2]
```diff
@@ -0,0 +1,21 @@
+// Copyright 2022 Meta Platforms, Inc. and affiliates.
```
Comment (u1f35c) [inline internal/context.go]: 2023

**Review by u1f35c — DISMISSED** [review]

File: context.go (hunk 1 above)
Comment (xaionaro) [reply context.go]: Addressed.

File: internal/context.go (hunk 2 above)
Comment (xaionaro) [reply internal/context.go]: Addressed.

Comment (xaionaro) [reply internal/context.go]: Addressed.

**Pull request merged**